Refactor CoNLL-related notebooks

frreiss opened the pull request:
This PR pulls parts 2, 3, and 4 of the CoNLL example into a new "tutorials" directory and adds a lot of descriptive text to part 1. I've also given the "part 1" notebook a more descriptive name, "Analyze Model Outputs".

BryanCutler commented on notebooks/Analyze_Model_Outputs.ipynb:
```diff
@@ -0,0 +1,3047 @@
+{
```
<p>nit: Dataframe -&gt; DataFrame</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

frreiss replied:
<p>Fixed now. Thanks for catching that!</p>

BryanCutler commented on notebooks/Analyze_Model_Outputs.ipynb:
```diff
@@ -0,0 +1,3047 @@
+{
```
<p>nit: dataframes, Dataframe -&gt; DataFrame</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

frreiss replied:
<p>Fixed.</p>

BryanCutler commented on notebooks/Analyze_Model_Outputs.ipynb:
```diff
@@ -0,0 +1,3047 @@
+{
```
<p>maybe also say that the line below uses <code>values</code>  to unwrap the TokenSpanArray from the Series?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

frreiss replied:
<p>Good point; there's quite a bit here that I didn't explain. I've expanded on this cell significantly in the next commit.</p>

BryanCutler commented on notebooks/Analyze_Model_Outputs.ipynb:
```diff
@@ -0,0 +1,3047 @@
+{
```
<p>corpus's -&gt; corpora ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

frreiss replied:
<p>I'm not sure what's the possessive form of "corpus" :)  I'll just change this to "corpus labels".</p>

BryanCutler commented on notebooks/Analyze_Model_Outputs.ipynb:
```diff
@@ -0,0 +1,3047 @@
+{
```
<p>let -&gt; lets</p><p>"performed least" -&gt; "performed the least" - I think..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

frreiss replied:
<p>This sentence was a mess. I've rewritten it as "We can use these per-document statistics to find documents where this model performed poorly."</p>

BryanCutler commented:
Looks great @frreiss , lots of good information added!  Were you going to elaborate about the last example with Santa Fe?

frreiss commented:
Thanks for the detailed review, @BryanCutler ! I have addressed the inline comments and added some additional content to the end to flesh out the "Santa Fe" example.

frreiss commented:
Related Github issue: #64
